linkerd/app/core/src/trace.rs

```diff
@@ -90,24 +117,26 @@ impl LevelHandle {
         let (_, handle) = with_filter_and_format("", "");
         handle
     }
+}
 
+impl LevelHandle {
     pub fn set_level(&self, level: impl AsRef<str>) -> Result<(), Error> {
         let level = level.as_ref();
         let filter = level.parse::<EnvFilter>()?;
         match self {
-            Self::Json(level) => level.reload(filter)?,
-            Self::Plain(level) => level.reload(filter)?,
+            LevelHandle::Json(level) => level.reload(filter)?,
+            LevelHandle::Plain(level) => level.reload(filter)?,
```

Comment:
Why change this from `Self`?

Reply:
I think these methods briefly moved to a different type and then came back. Will back that out.